instrumentation/spring/spring-webmvc-3.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/springwebmvc/HandlerMappingResourceNameFilter.java

```diff
@@ -121,7 +121,6 @@ public int getOrder() {
     public BeanDefinition() {
       setScope(SCOPE_SINGLETON);
       setBeanClass(HandlerMappingResourceNameFilter.class);
-      setBeanClassName(HandlerMappingResourceNameFilter.class.getName());
```

Comment:
Can you describe the change a bit? We don't lose anything by deleting the line?

Reply:
By the way I notice DD seems to take the opposite approach

https://github.com/DataDog/dd-trace-java/blob/master/dd-java-agent/instrumentation/spring-webmvc-3.1/src/main/java/datadog/trace/instrumentation/springweb/HandlerMappingResourceNameFilter.java#L92

Reply:
good point, I'll add a comment to the code tomorrow to explain why bean class is used instead of bean class name